cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -3,16 +3,17 @@ NAME
                         definition
 
 SYNOPSIS
-       bal grpc [--input <proto-file-path>] [--output <path>]
-                        [--mode] (client | service | proxy)
+       bal grpc [--input] <proto-file-path> [--output] <path>
```

Comment:
Is the `--input` optional? In that case, it should be `[--input <proto-file-path>]`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/ddaea2a9d5eecb9756bcc418a37a99ee8672aeda